public/creators-landing/src/components/img/brave-rewards-creators-logo.jsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+
+function Icon(props) {
```

Comment:
This could help save some code converting svgs to components:
https://www.npmjs.com/package/react-inlinesvg



Example:
```
      <SVG
        src={img/svg/icon.svg}
        width={128}
        height="auto"
        title="React"
      />

```

Reply:
Then you can just keep all your svg assets in a single folder

Reply:
I think this would be great to have in the future!